Edit user

ag3722 opened the pull request.

ag3722 committed:
User&MerchantAPI

- Integrated /getUser with Account Info page
- Integrated /getMerchant with Business Info page

ag3722 committed:
EditUserAPI

- Integrated /editUser with settings page to edit user details

ag3722 committed:
Token Comments Removed

Theveenan reviewed (dismissed):
The edit user page works fine, although the edit business details page doesn't load/edit data. Below is a screenshot of what I see when I click on the page for a user who is a merchant.


![image](https://github.com/user-attachments/assets/e8eb93b1-f784-4490-99fa-3e0972caea09)

SaifA21 reviewed (changes requested):
When a user tries to edit their business details, the existing value is not passed into the model like it is for personal settings

ag3722 committed:
Updated Business Info Page

- Fixed modal to display existing value when editing business details to match personal info page

ag3722 committed:
Merge branch 'staging' into editUser

SaifA21 reviewed (changes requested):
1 Small change request, lets make the text boxes wrap text or have larger height when editing details since it hard to make changes for bio or address 
![image](https://github.com/user-attachments/assets/1ebb4feb-a0d5-4456-b19f-f5acc2e8ce61)

ag3722 committed:
Fix Text Input Fields

- Fixed the text input boxes to dynamically adjust the height based on the text length

SaifA21 reviewed (approved):
LGTM
